docs/podman-run.1.md

```diff
@@ -279,12 +279,14 @@ The example maps gids 0-2000 in the container to the gids 30000-31999 on the hos
 
 Add additional groups to run as
 
-**--healthcheck-command**=*command*
+**--healthcheck-command**=*"command"* | *'["command", "arg1", ...]'*
 
 Set or alter a healthcheck command for a container. The command is a command to be executed inside your
 container that determines your container health.  The command is required for other healthcheck options
 to be applied.  A value of `none` disables existing healthchecks.
 
+You need to specify multi option commands in the form of a json string.
```

Comment:
Maybe reword to "Multiple options can be passed in the form of a JSON array; otherwise, the command will be interpreted as an argument to `/bin/sh -c`"

Reply:
Probably. I just copy&pasted the sentence from `--entrypoint` :-)